xcode/Subconscious/Shared/Components/Common/Profile/ProfilePicSm.swift

```diff
@@ -15,7 +15,7 @@ struct ProfilePicSmImage: View {
         image
             .resizable()
             .aspectRatio(contentMode: .fill)
-            .frame(width: 24, height: 24)
+            .frame(width: 32, height: 32)
```

Comment:
Looks better with the reworked Story views

Reply:
WFM.

Nit: One thought is that we should restrict ourselves to 2 (Sm, Lg), or max 3 (Sm, Md, Lg) pfp sizes throughout the app. This probably deserves to be an app constant.

Reply:
I realised I totally missed the existence of the Md size option!

I reabstracted a little bit and merged down to one component `ProfilePic` that accepts a `ProfilePicSize` enum with three options:

1. sm - 24px - backlinks, stacked avatar lists
2. md - 32px - story byline
3. lg - 48px - profile header